lib/bap_primus/bap_primus_memory.ml

```diff
@@ -304,4 +304,18 @@ module Make(Machine : Machine) = struct
     find_layer addr layers |>
     function Some {perms={readonly}} -> not readonly
            | None -> false
+
+  let fill base ~len value =
+    let filter_out values region =
```

Comment:
The word `filter` already assumes `out`, as there is no `filter_in` :) If you have trouble remembering what "filter" is doing, as I am, then just pick a better name that doesn't involve filter, e.g., `remove_all_mapped`.